core/services/ardupilot_manager/frontend/index.html

```diff
@@ -0,0 +1,212 @@
+                    </div>
+                    <div class="cell-1">
+                        <span class="mif-floppy-disk mif-lg d-flex flex-justify-center pr-2"></span>
+                    </div>
+                    <div class="cell-1">
+                        <span class="mif-lock mif-lg d-flex flex-justify-center pr-2"></span>
+                    </div>
+                    <div class="cell-1"></div>
+                </div>
+
+                <div class="row p-1 m-1 bg-gray">
+                    <div class="cell-2">
+                        <input type="text" data-role="input" data-clear-button="false" v-model="new_endpoint.name" />
+                    </div>
+                    <div class="cell-1">
+                        <input type="text" data-role="input" data-clear-button="false" v-model="new_endpoint.owner" />
+                    </div>
+                    <div class="cell-2">
+                        <select data-role="select" v-model="new_endpoint.connection_type" data-drop-full-size="true">
+                          <option v-for="(value, key) in connection_type_options" :value="key" :key="key">
+                            {{ value }}
+                          </option>
+                        </select>
+                    </div>
+                    <div class="cell-2">
+                        <input type="text" data-role="input" data-clear-button="false" v-model="new_endpoint.place" />
+                    </div>
+                    <div class="cell-2">
+                        <input type="text" data-role="input" data-clear-button="false" v-model="new_endpoint.argument" />
+                    </div>
+                    <div class="cell-1">
+                        <input type="checkbox" data-role="checkbox" v-model="new_endpoint.persistent" class="d-flex flex-justify-center" />
+                    </div>
+                    <div class="cell-1">
+                        <input type="checkbox" data-role="checkbox" v-model="new_endpoint.protected" class="d-flex flex-justify-center" disabled />
+                    </div>
+                    <div class="cell-1">
+                        <button class="button primary success cycle" @click="createEndpoint()">
+                          <span class="mif-plus"></span>
+                        </button>
+                    </div>
+                </div>
+
+                <div class="row p-1 m-1 bg-light flex-align-center" v-for="endpoint in registeredEndpoints" :key="endpoint.name">
+                    <div class="cell-2">
+                        <div>{{ endpoint.name }}</div>
+                    </div>
+                    <div class="cell-1">
+                        <div>{{ endpoint.owner }}</div>
+                    </div>
+                    <div class="cell-2">
+                        <div class="d-flex flex-justify-end">
+                            {{ connection_type_options[endpoint.connection_type] }}
```

Comment:
should this be a computed ?

Reply:
Yeah, sure.
It doesn't change the behavior, but but I also prefer to have all not-direct variables on computed.
Will do.